frappe/core/doctype/data_export/data_export.js

```diff
@@ -144,15 +144,28 @@ const get_doctypes = (parentdt) => {
 
 const add_doctype_field_multicheck_control = (doctype, parent_wrapper) => {
 	const fields = get_fields(doctype);
-
-	const options = fields.map((df) => {
+	const standard_fields = [
+		{
+			label: "Owner",
```

Comment:
FYI model.js has standard fields which can be used instead of hardcoding it here.

Reply:
I've made changes to the code to utilize the standard fields from model.js. However, a new issue has arisen where the exported titles are not consistent. The problem lies in exporter.py, where we've hardcoded the field variables for `owner and creation` since they are standard fields. To address this, I believe we should introduce a method in the Meta class that returns standard field meta / allow standard fields to be fetched within the same `get_field(fieldname)` method, condition would be like If `_fields.get` returns None, it can then search in the `standard_set_once_fields`. 

Kindly suggest on this.